src/sumo.erl

```diff
@@ -59,7 +59,7 @@
 -type field_attrs() :: [field_attr()].
 
 -type field_type() :: integer | float | string | binary | boolean
-                    | date | datetime | custom.
+                    | date | datetime | custom | text.
```

Comment:
It doesn't make sense. Take a look at: https://github.com/inaka/sumo_db/issues/251